### PR TITLE
Reputation fix

### PR DESCRIPTION
**Pull request by antonypap**
Renaming variables, random access and personalityMod fix

### CODE REVIEW

**Review by MattSScott — APPROVED** [review]
Nice changes. LGTM

File: pkg/infra/teams/team3/election_strat.go [hunk 1]
```diff
@@ -159,15 +159,23 @@ func (a *AgentThree) calcW2(id commons.ID) float64 {
...
-	ids := commons.ImmutableMapKeys(vAS)
+
+	// Create agent map deep copy
+	agentMap := make(map[commons.ID]state.HiddenAgentState)
+	itr := vAS.Iterator()
+	for !itr.Done() {
+		id, hiddenState, _ := itr.Next()
+		agentMap[id] = hiddenState
+	}
+	// ids := commons.ImmutableMapKeys(vAS)
```
Comment (MattSScott) [inline pkg/infra/teams/team3/election_strat.go]: Would make this `agentMapDeepCopy` so as not to shadow outer scope

**Review by MattSScott — APPROVED** [review]
Once minor recommendation, on reflection

**Comment by antonypap** [comment]
Somehow those changes have a huge implication on loot allocation...

**Comment by antonypap** [comment]
Hotfix: remove TSN from loot thresholds

File: pkg/infra/teams/team3/election_strat.go [hunk 2]
```diff
@@ -162,15 +162,23 @@ func (a *AgentThree) calcW2(id commons.ID) float64 {
...
-	ids := commons.ImmutableMapKeys(vAS)
+
+	// Create agent map deep copy
+	agentMapDeepCopy := make(map[commons.ID]state.HiddenAgentState)
+	itr := vAS.Iterator()
+	for !itr.Done() {
+		id, hiddenState, _ := itr.Next()
+		agentMapDeepCopy[id] = hiddenState
+	}
+	// ids := commons.ImmutableMapKeys(vAS)
```
Comment (MattSScott) [inline pkg/infra/teams/team3/election_strat.go]: As we discussed - easier to just use random shuffle on the `ids` array

**Review by MattSScott — COMMENTED** [review]
1 change, otherwise changes are good

**Comment by antonypap** [comment]
some agents might end up with the same "shuffled list", cause of the go routine and nanosecond accuracy of rand, but it should be fine

**Pull request closed without merging**